src/RemoteSymbolProvider/StadiaSymbolStoreSymbolProviderTest.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//#include <gmock/gmock.h>
+#include <absl/strings/str_format.h>
+#include <absl/strings/str_replace.h>
+#include <gtest/gtest.h>
+
+#include <QCoreApplication>
+#include <QPointer>
+#include <string_view>
+
+#include "Http/MockDownloadManagerInterface.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/Result.h"
+#include "OrbitBase/StopSource.h"
+#include "OrbitBase/VoidToMonostate.h"
+#include "OrbitGgp/MockClient.h"
+#include "QtUtils/MainThreadExecutorImpl.h"
+#include "RemoteSymbolProvider/StadiaSymbolStoreSymbolProvider.h"
+#include "Symbols/MockSymbolCacheInterface.h"
+#include "TestUtils/TestUtils.h"
+
+namespace orbit_remote_symbol_provider {
+
+namespace {
+using orbit_base::Canceled;
+using orbit_base::CanceledOr;
+using orbit_base::Future;
+using orbit_ggp::SymbolDownloadInfo;
+using SymbolDownloadQuery = orbit_ggp::Client::SymbolDownloadQuery;
+using orbit_symbol_provider::ModuleIdentifier;
+using orbit_test_utils::HasError;
+using orbit_test_utils::HasNoError;
+using ::testing::_;
+using ::testing::Return;
+
+const std::filesystem::path kSymbolCacheDir{"valid/symbol/cache/path"};
+constexpr std::string_view kOnlyAvailableModuleName{"only_available_module_name"};
+constexpr std::string_view kOnlyAvailableModuleBuildId{"ABCD12345678"};
+const ModuleIdentifier kValidModuleId{"module/path/to/only_available_module_name", "ABCD12345678"};
```

Comment:
Consider using `kOnlyAvailableModuleName` and `kOnlyAvailableModuleBuildId` in the construction of `kValidModuleId` to make the relationship clear.